metaci/build/test_models.py

```diff
@@ -0,0 +1,43 @@
+from django.test import TestCase
+from metaci.build.models import Build
+from metaci.conftest import (
+    RepositoryFactory,
+    BranchFactory,
+    PlanFactory,
+    PlanRepositoryFactory,
+    PlanScheduleFactory,
+)
+
+
+# Create your tests here.
+class BuildTestCase(TestCase):
+    def test_empty_build_init(self):
+        build = Build()
+        self.assertTrue(build)
```

Comment:
What's the value of this test?

Reply:
The virtue of having the test at all is to ensure that my \_\_init\_\_ code works when no arguments are passed.

The virtue of asserting that it didn't return null is very minor so I removed it in ceb06d4f34dd7dac72d91cee8108ffb7e91eaf96